src/ergw_aaa_diameter_srv.erl

```diff
@@ -146,9 +146,11 @@ await_response(Promise, Timeout) ->
 
 handle_plain_error(Module, Error, Request, SvcName, CallOpts)
   when is_atom(Module) ->
+    ?LOG(error, "DIAMETER error in ~p of request type ~p with: ~p", [Module, hd(Request), Error]),
     erlang:apply(Module, handle_error, [Error, Request, SvcName, undefined, CallOpts]);
 handle_plain_error([Module | Extra], Error, Request, SvcName, CallOpts)
   when is_atom(Module) ->
+    ?LOG(error, "DIAMETER error in ~p of request type ~p with: ~p", [Module, hd(Request), Error]),
```

Comment:
please change the log level to `debug` for now (agreed with Ferenc)

Reply:
Updated.